### ISSUE TITLE
error TS2366: Function lacks ending return statement and return type does not include 'undefined'.

### ISSUE THREAD

**Issue by GoMino** [reime005/react-native-camera-hooks#10]
Some typescript files seems to be picked up by tsc compiler:

```
node_modules/react-native-camera-hooks/src/recordVideo.ts:33:4 - error TS2366: Function lacks ending return statement and return type does not include 'undefined'.

33 ): Promise<RecordResponse> => {
      ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/react-native-camera-hooks/src/takePicture.ts:39:4 - error TS2366: Function lacks ending return statement and return type does not include 'undefined'.

39 ): Promise<TakePictureResponse> => {
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

It seems that the entry point is not correctly exposing the correct file:
`"main": "src/index.js" `
should probably be 
`"main": "dist/react-native-camera-hooks.js"`

**Comment by reime005** [comment]
oh yea sure, that could be. Shame that I never used this lib in a TS project. I'll look into that, thanks

**Comment by GoMino** [comment]
Great thanks!

Do you know when you plan to release a new version of the lib?

For now my workaround the issue is to manually import like this:
`import { useCamera } from 'react-native-camera-hooks/dist/react-native-camera-hooks';`

**Comment by reime005** [comment]
Should be fixed with v0.2.0. Please close this issue then
